server/utils/validation/loginSchema/index.js

```diff
@@ -0,0 +1,10 @@
+const { string, object } = require('yup');
+
+const loginSchema = object({
+  email: string().email('Must be a valid email').required('Email is required'),
+  password: string()
+    .min(8, 'Password must be at least 8 char')
+    .required('Password is required'),
```

Comment:
the yup error handler provaied the exact same massage so why are you writing it this way?